OpenRA.Game/SymbolicSupportDirectory.cs

```diff
@@ -0,0 +1,28 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+
+namespace OpenRA
+{
+	/// <summary>
+	/// Responsible for constants and common logic of dealing with the Symbolic support directory path.
+	/// </summary>
+	public class SymbolicSupportDirectory
```

Comment:
Please split these support directory changes into their own commit.
OpenRA conventions are to keep distinct changes in distinct commits!

Reply:
Ack'ed-  I'll separate this facet to a separate PR.